go/libraries/doltcore/merge/merge_prolly_rows.go

```diff
@@ -1078,12 +1056,111 @@ func (m *primaryMerger) merge(ctx *sql.Context, diff tree.ThreeWayDiff, sourceSc
 	case tree.DiffOpRightDelete:
 		return m.mut.Put(ctx, diff.Key, diff.Right)
 	case tree.DiffOpDivergentModifyResolved:
-		return m.mut.Put(ctx, diff.Key, diff.Merged)
+		// any generated columns need to be re-resolved because their computed values may have changed as a result of
+		// the merge
```

Comment:
Do we need this same logic for `secondaryMerge.merge`?

Reply:
Probably, I'll add a note